docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "dt_factor = apply_prefix(1, unit)\n",
+    "\n",
+    "# exp1.set_analysis_options(p0={\"amp\": 0.5, \"tau\": 20 * conversion_factor, \"base\": 0.5})\n",
+    "\n",
+    "expdata1 = exp1.run(backend=backend, shots=2000)\n",
+    "expdata1.block_for_results()  # Wait for job/analysis to finish.\n",
+    "\n",
+    "# Display the figure\n",
+    "display(expdata1.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DbAnalysisResultV1\n",
+      "- name: @Parameters_T2HahnAnalysis\n",
+      "- value: [4.73150237e-01 5.03648507e-01 1.98283007e-05] ± [5.15527131e-03 3.03978270e-03 5.77293057e-07]\n",
+      "- χ²: 0.7488240853426228\n",
+      "- quality: good\n",
+      "- extra: <4 items>\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n",
+      "DbAnalysisResultV1\n",
+      "- name: T2\n",
+      "- value: 1.9828300679956625e-05 ± 5.772930568055365e-07 s\n",
+      "- χ²: 0.7488240853426228\n",
+      "- quality: good\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print results\n",
+    "for result in expdata1.analysis_results():\n",
+    "    print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Providing initial user estimates\n",
+    "The user can provide initial estimates for the parameters to help the analysis process. Because the curve is expected to decay toward $0.5$, the natural choice for parameters $A$ and $B$ is $0.5$.In this experiment, `t2hahn` is the parameter of interest. Good estimate for it is the value computed in previous experiments on this qubit or a similar value computed for other qubits."
```

Comment:
The choice for A is rightfully 0.5, but it's not because the curve decays toward 0.5

Reply:
changed to:
`Because the curve is expected to decay toward  0.5 , the natural choice for parameter  𝐵  is  0.5 . Since if there was no delay we would expect that the probability to measure 1 is  100% , so we will guess that A is  0.5 .`